### PR TITLE
Mjw/blob auto creation

### PR DESCRIPTION
**Pull request by mwevill**

### CODE REVIEW

File: templates/storage-container.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "publicAccess": {
+      "type": "string",
+      "defaultValue": "none",
+      "allowedValues": [
+        "Container",
+        "Blob",
+        "None"
+      ],
+      "metadata": {
+        "description": "Level of publicly accesability of the created container"
```
Comment (jack-education) [inline templates/storage-container.json]:
```suggestion
        "description": "Level of publicly accessibility of the created container"
```

File: templates/storage-container.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    }
+  },
+  "variables": {},
+  "resources": [
+    {
+      "name": "[concat(parameters('storageAccountName'), '/default/', parameters('containerName'))]",
+      "type": "Microsoft.Storage/storageAccounts/blobServices/containers",
+      "apiVersion": "2018-07-01",
+      "properties": {
+        "publicAccess" : "[parameters('publicAccess')]"
```
Comment (jack-education) [inline templates/storage-container.json]:
```suggestion
        "publicAccess" : "[parameters('publicAccess')]",
        "metadata" : "[parameters('metadata')]"
```

Comment (mwevill) [reply templates/storage-container.json]: Metadata is optional I see no need for it. No example I have seen has used it

File: templates/storage-container.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    "publicAccess": {
+      "type": "string",
+      "defaultValue": "none",
+      "allowedValues": [
+        "Container",
+        "Blob",
+        "None"
+      ],
+      "metadata": {
+        "description": "Level of publicly accessibility of the created container"
```
Comment (jack-education) [inline templates/storage-container.json]:
```suggestion
        "description": "Level of public accessibility of the created container"
```

**Pull request merged**